### PR TITLE
chore: add dynamic wrapper for the catalog error logs module

### PR DESCRIPTION
**Pull request by Zaperex**
## Description

Adds the [@backstage/plugin-catalog-backend-module-logs](https://github.com/backstage/backstage/tree/master/plugins/catalog-backend-module-logs) plugin to improve RHDH serviceability  

Also reverted the `backstage` metadata in the `package.json` of the dynamic plugin wrappers to match the format used in the upstream.
## Which issue(s) does this PR fix

- Fixes [RHIDP-3125](https://issues.redhat.com/browse/RHIDP-3125)

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

File: dynamic-plugins.default.yaml [hunk 1]
```diff
@@ -872,4 +872,6 @@ plugins:
     pluginConfig:
       catalog:
         providers:
-          microsoftGraphOrg: {}
\ No newline at end of file
+          microsoftGraphOrg: {}
+  - package: ./dynamic-plugins/dist/backstage-plugin-catalog-backend-module-logs-dynamic
```
Comment (nickboldt) [inline dynamic-plugins.default.yaml:876]:
```suggestion

  - package: ./dynamic-plugins/dist/backstage-plugin-catalog-backend-module-logs-dynamic
```

File: dynamic-plugins/wrappers/backstage-plugin-catalog-backend-module-logs/package.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": true,
+  "publishConfig": {
+    "access": "public"
+  },
+  "backstage": {
+    "role": "backend-plugin-module",
+    "supported-versions": "1.29.2",
+    "pluginId": "catalog",
```
Comment (nickboldt) [inline dynamic-plugins/wrappers/backstage-plugin-catalog-backend-module-logs/package.json:14]:
```suggestion
    "pluginId": "backstage-plugin-catalog-backend-module-logs",
```

File: dynamic-plugins/wrappers/backstage-plugin-catalog-backend-module-msgraph-dynamic/package.json [hunk 3]
```diff
@@ -12,7 +12,7 @@
     "role": "backend-plugin-module",
     "supported-versions": "1.27.7",
     "pluginId": "catalog",
```
Comment (nickboldt) [inline dynamic-plugins/wrappers/backstage-plugin-catalog-backend-module-msgraph-dynamic/package.json:14]:
```suggestion
    "pluginId": "backstage-plugin-catalog-backend-module-msgraph",
```

**Comment by nickboldt** [comment]
Probably OK. Looks like you re-ran the backstage-cli command to cleanup the repo, as there are more files in this PR can related to the catalog error log module... 

```
./node_modules/@backstage/cli/bin/backstage-cli repo fix --publish
```

**Comment by Zaperex** [comment]
oh I believe I added those on purpose to update the backstage metadata for all the other dynamic plugin wrappers to match the upstream backstage metadata format.

**Pull request merged**